src/lib/strapi.js

```diff
@@ -0,0 +1,13 @@
+const { STRAPI_URL, STRAPI_TOKEN } = import.meta.env;
+
+export async function fetchPosts() {
```

Comment:
this should say `fetchArticles`, but you get the point. It'll all be replaced when it's fully fledged anyways